src/transforms.coffee

```diff
@@ -395,6 +395,9 @@ module.exports.makeYieldConditionally = makeYieldConditionally = (simpleLoops, w
         else
           autoYieldStmt = ""
         node.update "var __gen#{node.left.source()} = #{node.right.source()}; while (true) { var __result#{node.left.source()} = __gen#{node.left.source()}.next(); if (__result#{node.left.source()}.done) { #{node.left.source()} = __result#{node.left.source()}.value; break; } var _yieldValue = __result#{node.left.source()}.value; if (this.onAetherYield) { this.onAetherYield(_yieldValue); } yield _yieldValue; #{autoYieldStmt} }"
+      else if node.right.callee.type is S.MemberExpression and node.originalNode.callee?.property?.name in ["forEach", "map"] and node.right.arguments.length
```

Comment:
Currently, the built-in methods that accept generatorified function callbacks' names are hard-coded. It would be better to get them from innerator, somehow.